resources/views/admin/_users.blade.php

```diff
@@ -8,18 +8,20 @@
               <th width="20%">First Name</th>
               <th width="20%">Last Name</th>
               <th width="21%">Signed In</th>
-              <th width="12%">Admin</th>
+              <th width="9%">Admin</th>
+              <th width="9%">Inactive</th>
```

Comment:
The sum of the percentages is greater than 100%, no?

Reply:
Would it be better to clean up this table so it doesn't rely on hard-coded width percentages, like you did when you were refactoring some tables some months back?

Reply:
i can do that!